src/sass/layouts/_header.scss

```diff
@@ -1,19 +1,83 @@
 .header {
-  max-width: $mobile-width;
-  padding: 24px 0 21px 0;
+  padding: 19px 0;
   margin: 0 auto;
   border-bottom: 1px solid rgba(0, 0, 0, 0.16);
+  @include media-desktop {
+    padding: 48px 0 42px 0;
+  }
 
   &__container {
     @include flex(row, space-between, center, nowrap);
   }
 
+  &__logo {
+    flex-shrink: 3;
+    @include flex(row, center, center, nowrap);
+  }
+
+  &__hamburger {
+    @include media-desktop {
+      display: none;
+    }
+  }
+
   &__navigation {
     display: none;
+    @include media-desktop {
+      display: block;
+      flex-shrink: 2;
+    }
   }
 }
 
-.logo__img {
+.navigation {
+  margin: 0;
+  @include flex(row, space-between, center, wrap);
+  gap: 36px;
+
+  &__item {
+    list-style-type: none;
+    position: relative;
+    z-index: 0;
+    &:hover {
+      &:after {
+        content: '';
+        position: absolute;
+        background-color: $color-dark;
+        width: 110%;
+        height: 2px;
+        bottom: -54px;
+        left: 50%;
+        transform: translate(-50%, 50%);
+        z-index: 1;
+      }
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $color-dark;
+    @extend %font-tertiary;
+    @include font(normal, 500, 2.0rem, 1.4rem);
+    text-transform: uppercase;
+  }
+}
+
+.logo__ref {
+  @include flex(row, center, center, nowrap);
   width: 217px;
   height: 20px;
+  @include media-desktop {
+    width: 396px;
```

Comment:
how does it work? this will result in the following, right?
```css
@media (min-width: 375px) {
  .logo__ref {
    width: 396px?
  }
}
```
how will the page look at `380px`? will it have horizontal scroll?